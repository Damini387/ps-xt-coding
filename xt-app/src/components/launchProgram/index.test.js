import React from 'react';
import { shallow } from 'enzyme';
import LaunchProgram from './index';
import Filters from '../filters/index';
import Rockets from '../rockets/index';

let wrapper = shallow(
    <LaunchProgram />);

describe('Launch Program', () => {

	it('should render the Launch Program Component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});

	it('should render Filters and Rockets component correctly', () => {
        expect(wrapper.find(Filters)).toHaveLength(1);
        expect(wrapper.find(Rockets)).toHaveLength(1);
	});

});