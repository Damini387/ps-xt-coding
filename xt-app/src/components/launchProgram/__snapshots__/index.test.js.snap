// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Launch Program should render the Launch Program Component correctly 1`] = `
<Fragment>
  <h2>
    SpaceX Launch Program
  </h2>
  <div
    className="launchPad"
  >
    <Connect(Filters)
      filterRockets={[Function]}
    />
    <Rockets />
  </div>
</Fragment>
`;
